www/js/charts/histogramChartDirective.js

```diff
@@ -163,11 +171,13 @@ export default ($log) => {
                     .range([barContainerHeight, 0]);
 
             let yFrequency = scaleLinear()
-                    .domain([0, max(frequency)])
+                    .domain([0, domainMax])
                     .range([barContainerHeight, 0]);
             // Axises
             xAxis = axisBottom(x);//.tickValues(leadtime);
-            let yAxisLeft = axisLeft(yFrequency).ticks(ticks).tickSize(-barContainerWidth);//.ticks(ticks).tickSize(-barContainerWidth);
+            let yAxisLeft = axisLeft(yFrequency)
+                .tickValues(range(0, domainMax + 1, domainMax / 5))
```

Comment:
https://github.com/mbostock/d3/wiki/Arrays#d3_range

range(start, stop, step). stop is an exclusive limit.